### PR TITLE
Add timeout configuration

### PR DESCRIPTION
**Pull request by corbinschwartz**

### CODE REVIEW

**Comment by corbinschwartz** [comment]
@shystruk I figured this could be helpful. Thoughts?

File: src/no-internet.js [hunk 1]
```diff
@@ -99,6 +102,10 @@
             callback(OFFLINE);
         };
 
+        xhr.ontimeout = function (e) {
```
Comment (shystruk) [inline src/no-internet.js:105]: please remove not used param 😄

**Comment by shystruk** [comment]
Hi, 
Thank you for contribution.
Good idea. 
I'm wondering about a slow connection. It should not say you are offline, right?

**Comment by shystruk** [comment]
@corbinschwartz 
Just faced that issue on Lyft app. I requested a line and could not see a car on a map (connection was slow), so driver cancel the ride 😄

**Comment by corbinschwartz** [comment]
@shystruk, thanks for merging :) My bad about the unused param, good catch.

**Pull request merged**